app/lib/package.rb

```diff
@@ -40,6 +42,7 @@ def build_zip
 
     Zip::File.open("#{@directory}/#{@translation.zip_name}", Zip::File::CREATE) do |zip_file|
       add_pages(zip_file, manifest)
+      add_tips(zip_file, manifest)
```

Comment:
`add_tips` should be gated on the `include_tips` resource language attribute. we only include the tips xml if the attribute is true